extensions/stackdriver/metric/registry.cc

```diff
@@ -125,6 +125,11 @@ StackdriverOptions getStackdriverOptions(
                                        grpc::InsecureChannelCredentials());
     options.metric_service_stub =
         google::monitoring::v3::MetricService::NewStub(channel);
+  } else if (!stub_option.monitoring_endpoint.empty()) {
+    auto channel = ::grpc::CreateChannel(stub_option.monitoring_endpoint,
```

Comment:
So this is either default creds or sts?
This is a good first step but I think we need both.

Reply:
Done.